xdsl/interactive/app.py

```diff
@@ -140,12 +131,8 @@ def compose(self) -> ComposeResult:
 
     def on_mount(self) -> None:
         """Configure widgets in this application before it is first shown."""
-
-        # register's the theme for the Input/Output TextArea's
-        self.input_text_area.register_theme(InputApp._DEFAULT_THEME)
-        self.output_text_area.register_theme(InputApp._DEFAULT_THEME)
-        self.input_text_area.theme = "my_theme_design"
-        self.output_text_area.theme = "my_theme_design"
+        self.input_text_area.theme = "vscode_dark"
```

Comment:
This is unrelated to the preloading of the text, could you please split out the two PRs? They also don't seem like they would need to be in a stack as they don't functionally depend on anything not in main

Reply:
Yes